src/engagement_db_to_analysis/engagement_db_to_analysis.py

```diff
@@ -2,28 +2,30 @@
 
 from core_data_modules.logging import Logger
 from core_data_modules.util import TimeUtils
+from core_data_modules.traced_data import TracedData, Metadata
 
 from src.engagement_db_to_analysis.cache import AnalysisCache
+from src.engagement_db_to_analysis import Filters
 
 log = Logger(__name__)
 
 
 def serialise_message(msg):
-
     msg = msg.to_dict()
-    msg["timestamp"] = TimeUtils.datetime_to_utc_iso_string(msg["timestamp"])
-    msg["last_updated"] = TimeUtils.datetime_to_utc_iso_string(msg["last_updated"])
+    msg["timestamp"] = msg["timestamp"].isoformat()
+    msg["last_updated"] = msg["last_updated"].isoformat()
 
     return msg
 
-def get_project_messages_from_engagement_db(dataset_configurations, engagement_db, cache_path=None):
+
+def _get_project_messages_from_engagement_db(analysis_configurations, engagement_db, cache_path=None):
```

Comment:
It is not clear that `cache_path` set to None while in the main `engagement_db_to_analysis.py` `engagement_db_datasets_cache_dir` is a required argument.

Reply:
Thanks for catching it, it was forgotten here after a previous refactor. Removed thanks.